Source/SIMPLib/CoreFilters/Testing/Cxx/ArrayCalculatorTest.cpp

```diff
@@ -53,13 +53,32 @@
 #include "SIMPLib/Testing/UnitTestSupport.hpp"
 
 #include "SIMPLib/CoreFilters/util/CalculatorOperator.h"
+#include "SIMPLib/DataContainers/DataContainerArray.h"
+#include "SIMPLib/DataContainers/DataContainer.h"
 
 class DummyObserver : public Observer
 {
 public:
   DummyObserver() = default;
   ~DummyObserver() override = default;
-  SIMPL_TYPE_MACRO(DummyObserver)
+  /**
+   * @brief Returns the name of the class for DummyObserver
+   */
+  /**
```

Comment:
Something odd happened here, and you've got duplicate comment blocks for this method.